### PR TITLE
Use CSS to show/hide search results

### PR DESCRIPTION
**Pull request by zackhall**
Diff:
* Use CSS to show/hide search results instead of adding/removing from DOM
* Add transition effects when showing/hiding results

### CODE REVIEW

File: assets/js/docs.js [hunk 1]
```diff
@@ -78,30 +79,40 @@
...
+              'visible': search(collection.title, searchValue),
+              'functions': _.map(collection.functions, function(data) {
+                data.visible = search(collection.title, searchValue) || search(data.name, searchValue);
+                return data;
+              })
+            };
+
+            // The collection is visible if searchValue matches the title
+            // or if there are visible functions
+            result.visible =
```
Comment (zackhall) [inline assets/js/docs.js]: @jdalton Would love some feedback on how to make this cleaner. Don't like assigning twice.

Comment (jdalton) [inline assets/js/docs.js]: No worries. I'll kick it around.

**Comment by jdalton** [comment]
Woo woo fancy transitions ‼️

**Comment by jdalton** [comment]
Thank you

**Pull request merged**